import_email_events.py

```diff
@@ -0,0 +1,17 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL,
+    locale VARCHAR(40),
+    domain VARCHAR(40),
+    template VARCHAR(64),
+    flow_id VARCHAR(64)
```

Comment:
> can flow_id be NOT NULL too

Yea the change/reset password emails might not have `flow_id` field. @rfk @philbooth I think the most obvious for distkey would be `flow_id`, ill update that.